### ISSUE TITLE
**fatal client is offline

### ISSUE THREAD

**Issue by sarya0113** [theupdateframework/notary#1371]
I am trying to configure the DTR on the ucp cluster. But at the time of configuring notary into the UCP cluster i am facing the below error:
 root@notary-VirtualBox:~# notary init -D 172.30.70.24/ritesh123/signer
DEBU[0000] Using the following trust directory:  /root/.docker/trust
**ERRO[0000]** could not reach https://172.30.70.24:443: Get https://172.30.70.24:443/v2/: x509: cannot validate certificate for 172.30.70.24 because it doesn't contain any IP SANs
INFO[0000] continuing in offline mode

If any one have any solution kindly response on the same.

**Comment by justincormack** [comment]
That error message suggests your certificate has a name not an IP address in the cert. You may be best off talking to the UCP support team.

**Comment by sarya0113** [comment]
Thank you justin for the reply. So if i am not wrong are you asking to check the DNS resolution is happening or not between UCP and DTR?

**Comment by justincormack** [comment]
No, you need to use the DNS name for the notary server not the IP address, because the certificate os only valid for a name.

**Comment by sarya0113** [comment]
So, DNS entry for my DTR hosting node is required in the DNS server right?

**Comment by justincormack** [comment]
Yes.

**Comment by sarya0113** [comment]
Thanks a lot Justin!
